velox/external/duckdb/modify_include_guards.sh

```diff
@@ -0,0 +1,10 @@
+echo $1
+TAGS=$(sed -n 's/^#ifndef \(_[A-Z][A-Z_]*_\)/\1/p' ${1})
```

Comment:
We only want to substitute the Thrift headers, so should we match `_THRIFT_` too?